machine/plexxi/px_switch2/i2ctools/i2ctools-plexxi.patch

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
I thought I made this comment on the first PR, but looking back I do not see it.  Perhaps I forgot to click the "Comment" button....

Anyway -- Since March 16, 2015 (over a year ago) the ONIE project no longer accepts i2ctools patches for EEPROM access.  That announcement and reasons are described here:

http://lists.opencompute.org/pipermail/opencompute-onie/2015-March/000512.html

We will only put our limited resources into updating and maintaining the busybox version of the onie-syseeprom command going forward.

That email describes how to use busybox for EEPROM access instead of i2ctools.